src/display/display.rs

```diff
@@ -68,10 +71,16 @@ pub(crate) struct Display<'a, N: Network, E: Environment> {
     server: Server<N, E>,
     tabs: TabsState<'a>,
     tick_rate: Duration,
+    logs: Logs,
 }
 
 impl<'a, N: Network, E: Environment> Display<'a, N, E> {
-    pub fn start(server: Server<N, E>) -> Result<()> {
+    pub fn start(server: Server<N, E>, verbosity: u8) -> Result<()> {
+        // Initialize the log channel.
+        let (log_sender, log_receiver) = mpsc::channel(1024);
```

Comment:
this number is probably too large, and entries above the log display limit will currently get truncated (they could potentially all be scrolled through, though)